x/wasm/keeper/proposal_handler.go

```diff
@@ -58,18 +56,8 @@ func handleStoreCodeProposal(ctx sdk.Context, k types.ContractOpsKeeper, p types
 	if err != nil {
 		return sdkerrors.Wrap(err, "run as address")
 	}
-	codeID, err := k.Create(ctx, runAsAddr, p.WASMByteCode, p.InstantiatePermission)
-	if err != nil {
-		return err
-	}
-
-	ourEvent := sdk.NewEvent(
-		sdk.EventTypeMessage,
-		sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
-		sdk.NewAttribute(types.AttributeKeyCodeID, fmt.Sprintf("%d", codeID)),
-	)
-	ctx.EventManager().EmitEvent(ourEvent)
-	return nil
+	_, err = k.Create(ctx, runAsAddr, p.WASMByteCode, p.InstantiatePermission)
```

Comment:
good to remove message.
no gov-result event here? (not that I see the need for it)

Reply:
The event is only emitted when we interact with a contract as the result data would not be available otherwise.